### PR TITLE
doc: Doc updates

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc updates to fix version number and build command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ west update
 To build the application, run the following command:
 
 ```shell
-west build -b $BOARD app
+west build -b $BOARD ncs-example-application/app
```
Comment (gmarull) [inline README.md:58]: this is not correct, user is expected to execute from the repository root

**Review by gmarull — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (de-nordic) [reply README.md:58]: Where is the expectation stated? It seems that README does not state that user is required to have previous knowledge on building Zephyr products; if user follows the README, building is not possible.

Comment (umapraseeda) [reply README.md:58]: @de-nordic

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This repository is versioned together with the [nRF Connect SDK main tree][sdk-n
 means that every time that nRF Connect SDK is tagged, this repository is tagged as well
 with the same version number, and the [manifest](west.yml) entry for `zephyr`
 will point to the corresponding nRF Connect SDK tag. For example, the `ncs-example-application`
-v2.3.0 will point to nRF Connect SDK v2.3.0. Note that the `main` branch always
+v2.4.0 will point to nRF Connect SDK v2.4.0. Note that the `main` branch always
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply README.md:21]: Yes, so may be we leave it as is? @de-nordic, @carlescufi?

File: README.md (hunk 1 above)
Comment (gmarull) [reply README.md:58]: then we need to add an extra `cd`, or modify `cd my-workspace` to `cd my-workspace/app`

Comment (de-nordic) [reply README.md:58]: User is already at my-workspace from previous step, but app is not. app is at lower level which is ncs-example-application.
I went through the not modified procedure and have recreated the dir structure.

**Comment by gmarull** [comment]
Btw, we need to sync with upstream example-application, it contains a few updates. @carlescufi

**Pull request closed without merging**